src/main/routes.ts

```diff
@@ -340,10 +340,13 @@ export class Routes {
      */
     app.get(PAYMENT_GATEWAY_ENTRY_URL, errorHandler(PaymentHandler));
     app.get(PAYMENT_RETURN_URL_CALLBACK, errorHandler(PaymentValidationHandler));
-
-    app.get('/api/v1/session', (req, res) => res.json(req.session));
+    app.get('/api/v1/session', (req, res) => {
```

Comment:
This implementation is not the right approach. You need to make sure the page is responding fast and not load and timeout.

Reply:
Changed